gradio/blocks.py

```diff
@@ -1362,10 +1362,10 @@ def handle_streaming_outputs(
                 if run not in self.pending_streams[session_hash]:
                     self.pending_streams[session_hash][run] = defaultdict(list)
                 self.pending_streams[session_hash][run][output_id].append(stream)
-                data[i] = {
-                    "name": f"{session_hash}/{run}/{output_id}",
-                    "is_stream": True,
-                }
+                if data[i]:
+                    data[i]["is_file"] = False
```

Comment:
Preserve the `orig_name` so that clients have an easier time downloading the byte stream

Reply:
But isn't ["name"] assigned to the same value in the next line?

Reply:
Yea we're using "name" to mean the url from which to download the stream so I'm keeping that the same. Just preserving `orig_name` (if it's there) so that we can give a meaningful name when we go to download the byte stream to a file.